tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/manifest.ini

```diff
@@ -0,0 +1,3 @@
+[parent:../manifest.ini]
+
+[test_settings_personalization.py]
```

Comment:
I'd rather see all the test files you are planning to add here, in the manifest.ini file, but then commented out, if you want them to not run, currently.